src/plugins/visualize/public/application/components/visualize_top_nav.tsx

```diff
@@ -234,7 +278,7 @@ const TopNav = ({
       appName={APP_NAME}
       setMenuMountPoint={setHeaderActionMenu}
       indexPatterns={indexPatterns}
-      showSearchBar
+      showSearchBar={TopNavMenuItemRenderType.IN_PORTAL}
```

Comment:
Have you tested that embed mode works as expected?

Reply:
I tested the embedded flow by saving the visualizations and adding them to a dashboard. Let me know if there is any other way we need to test the embed mode.